### PR TITLE
added letter/score db

### PR DESCRIPTION
**Pull request by DKear**
## What does this implement/fix? Explain your changes.
Adds the letter/score db. The endpoints are in ScoreRestController.
## Does this reference any currently open issues?
linking DB and Engine
## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have linted my code and I have run tests locally that pass
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation

## Any other comments?
Hopefully I've done this correctly, finally.

### CODE REVIEW

File: Database/GameDB/src/main/java/com/csc480red/gamedb/Score.java [hunk 1]
```diff
@@ -0,0 +1,187 @@
+package com.csc480red.scoresdb;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+public class Score {
```
Comment (BrendanOswego) [inline Database/GameDB/src/main/java/com/csc480red/gamedb/Score.java]: Please abstract this into some sort of Tile class where each Tile has a letter and a Score, and load all the tiles from the database from a letter table (i.e TileTable)

Comment (BrendanOswego) [reply Database/GameDB/src/main/java/com/csc480red/gamedb/Score.java]: such as this.
```java
public class Tile implements Serializable {
	
	private final char letter;
	private final int value;

	public class Tile(char letter, int value) {
		this.letter = letter;
		this.value = value;
	}

	public char getLetter() {
		return letter;
	}

	public int getValue() {
		return value;
	}
}
```

Comment (BrendanOswego) [reply Database/GameDB/src/main/java/com/csc480red/gamedb/Score.java]: Another way of looking at it is with a HashMap like:
```java
public class Score {
	
	private Map<String, Integer> data;

	public Score() {
		data = new HashMap<>();
	}

	public Score(Map<String, Integer> data) {
		this.data = data;
	}

	public void add(String letter, Byte value) {
		data.put(letter, value);
	}

	public HashMap<String, Byte> getData() {
		return data;
	}
}
```

**Comment by landonp1203** [comment]
Is this the same thing as PR #118 just in a new PR? @DKear

**Comment by DKear** [comment]
@landonp1203 Sorry, I never saw your comment. Yes it is, except I screwed up where I put the files so everything added to the gamedb directory has just been turned into delete.txt variants because I'm not sure how to edit that part of the pull request so this should make it so it doesn't break anything.

**Comment by landonp1203** [comment]
@DKear I will fix the issues

**Comment by landonp1203** [comment]
@DKear I fixed the problems, please next time make sure that the changes you are making are in the correct places so that there are no extra commits that should not be there.

You need to sync your local copy with the upstream so that you can update your fork accordingly.  Please make sure you are on the `dev` branch before you do any of this.  Run the following commands to update your local copy:

```
git fetch upstream
git reset --hard upstream/dev
git push -f origin dev
```

**Comment by landonp1203** [comment]
I also put the server on port 8092

**Pull request closed without merging**